src/detect-hostbits.c

```diff
@@ -1131,103 +1070,6 @@ static int HostBitsTestSig07(void)
     PASS;
```

Comment:
Is there a reason this test 07 was not removed? You mentioned in your s-v PR that this is converted too. I also see some rules w `set` `unset` commands..